src/main/java/com/scrive/pdftools/tools/flattener/LambdaFlattener.java

```diff
@@ -3,27 +3,25 @@
 import com.itextpdf.text.DocumentException;
 import com.itextpdf.text.exceptions.BadPasswordException;
 import com.itextpdf.text.pdf.PdfReader;
-import com.itextpdf.text.pdf.PdfStamper;
+import com.scrive.pdftools.tools.utils.PdfStamperAutoclosable;
 
 import java.io.ByteArrayInputStream;
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
-import java.io.OutputStream;
 
 public class LambdaFlattener {
 
-    public static ByteArrayOutputStream execute(FlattenerSpec flatSpec) throws IOException, DocumentException {
-        ByteArrayOutputStream bos = new ByteArrayOutputStream();
-        try (ByteArrayInputStream file = new ByteArrayInputStream(flatSpec.getFileContent())) {
+    public static byte[] execute(FlattenerSpec flatSpec) throws IOException, DocumentException {
+        try (ByteArrayInputStream file = new ByteArrayInputStream(flatSpec.getFileContent()); ByteArrayOutputStream bos = new ByteArrayOutputStream()) {
```

Comment:
It would be more readable to have each resource on a separate line
```suggestion
        try (ByteArrayInputStream file = new ByteArrayInputStream(flatSpec.getFileContent()); 
              ByteArrayOutputStream bos = new ByteArrayOutputStream()) {
```

Reply:
Yes you're right. Fixed